pkg/app/app.go

```diff
@@ -36,11 +49,28 @@ func CreateApp() App {
 		os.Exit(1)
 	}
 
+	app.log.Debug("Connecting Dao")
+	if app.dao, err = dao.NewDao(app.config.Dao, app.log.Logger); err != nil {
+		app.log.Error("Failed to initialize Dao\n")
+		app.log.Error(err.Error())
+		os.Exit(1)
+	}
+
+	app.log.Debug("Connecting Registry")
+	if app.registry, err = ansibleapp.NewRegistry(
+		app.config.Registry, app.log.Logger,
+	); err != nil {
+		app.log.Error("Failed to initialize Dao\n")
+		app.log.Error(err.Error())
+		os.Exit(1)
+	}
+
 	return app
 }
 
 func (a *App) Start() {
-	a.log.Info("Starting application")
+	a.log.Notice("Ansible Service Broker Started")
+	a.registry.LoadApps()
```

Comment:
Good catch; I actually should remove this line for this PR. I'm picturing this start method being the one responsibly for starting the http mux listener. That will call into methods like `broker.Bootstrap()` (not written yet), which *in turn*, will be the guy that calls `registry.LoadApps()`.

So the full story has yet to be written here. This PR should be a lot of support without action in `Start()`. I'll try to make a note of this in `//TODOs` to explain myself a little better inline.